tools/plugins/com.liferay.ide.upgrade.plan.core/src/com/liferay/ide/upgrade/plan/core/internal/UpgradePlannerService.java

```diff
@@ -311,6 +383,44 @@ private File _getUpgradePlannerStorageFile() throws IOException {
 		return file;
 	}
 
+	private StandardUpgradePlan _initUpgradePlan(IMemento upgradePlanMemento) {
```

Comment:
use UpgradePlan interface instead

Reply:
move it to interface means it should be set public and it will cuase other _loadUpgradeProblems and _loadUpgradeSteps should be set deault too.  I think we should not upgradePlanner interface should invole IMemento. What do you think it?